README.md

```diff
@@ -100,32 +106,37 @@ rss.device.monitor.enabled false
 
 EXAMPLE: HA cluster
 ```properties
+# used by client and worker to connect to master
+celeborn.master.endpoints=clb-1:9097,clb-2:9098,clb-3:9099
```

Comment:
on the client side, including worker, single and ha modes share this configuration.